### DISCUSSION TITLE
Video corruption between sddm login and LXQt desktop appearance

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2086] [Q&A] [answered]
On both of my desktops, I am getting some video corruption. This occurs between the sddm login and the point where the LXQt desktop appears on each. This is on Debian 11 (amd64, bullseye), upgraded from Debian 10 (buster) and also occurred under 10/buster.

Both desktops have AMD Radeon GPU's. Desktop 1's GPU is an on-board chip. Desktop 2's GPU is on a video card. The lspci command shows both are correctly using the _radeon_ driver and all AMD/Radeon-related packages, including xorg;s and firmware are installed, including firmware-amd-graphics. firmware-linux, firmware-linux-free,  firmware-linux-nonfree and firmware-misc-nonfree. 

Desktop 1 (VGA connection): 01:05.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] RS780L [Radeon 3000] (prog-if 00 [VGA controller])

Desktop 2 (HDMI connection): 02:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Cedar [Radeon HD 5000/6000/7350/8350 Series] (prog-if 00 [VGA controller])

I am attaching photos of the monitors from each, showing the corruption at the same point during the login process. Is this being caused by LXQt, or perhaps is it a limitation of the GPU's? If it's LXQt, is there a setting somewhere I should look at? The blue image is from Desktop 1, the grey image is from Desktop 2.

Thank you in advance.

![desktop1](https://user-images.githubusercontent.com/73129450/136584374-38eaff81-637c-4d5b-89cd-f520fd752710.jpg)
![desktop2](https://user-images.githubusercontent.com/73129450/136584754-f1dfff09-60e8-4e91-bc0b-20fa30885476.jpg)

**Comment by tsujan** [comment, marked as answer]
> Is this being caused by LXQt

Such problems absolutely don't have anything to do with LXQt. Check your compositor, graphic driver, etc.

**Reply by stefonarch** [reply]
On debian I've no such issues with

```
33:02:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Picasso (rev c4)
34-     Subsystem: ASUSTeK Computer Inc. Picasso
35-     Kernel driver in use: amdgpu

```
My suspect: radeon

**Reply by edwardp3** [reply]
I went into Preferences/Window Manager Tweaks (The window manager is xfwm4.), unchecked Enable Display Compositing in the Compositor section, then rebooted. During the reboot, the screen went from the sddm login straight into the LXQt desktop, no video corruption in between.

**Reply by tsujan** [reply]
I'm not familiar with xfwm4 but, maybe, it has an option that makes its compositor work with your graphic settings. Xfce forums may have an answer.

BTW, rebooting isn't needed; logging out and in is enough.

**Reply by stefonarch** [reply]
I heard of people using picom (or compton) with xfwm4, never tried, but openbox with those works fine for me.

**Reply by edwardp3** [reply]
I don't think openbox provides compositing, but with xfwm4, once I turned off the compositing, I lost the translucency in the menus and panel. Turning that back on, restored it.

**Reply by stefonarch** [reply]
Openbox itself  not, but as said pico  or compton are working fine with and as I heard also with xfwm4 as external compositor.

**Reply by edwardp3** [reply]
I've decided to continue using xfwm4 as the window manager, as I like the transparency it provides. The video corruption isn't really harming anything, so I can live with that.

Thank you for all the responses.
